### PR TITLE
Mayhem Heroes Integration

### PR DESCRIPTION
**Pull request by capuanob**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Build Stage
```
Comment (xansec) [inline Dockerfile:1]: Please do not use git clone or `ldd` in your Dockerfile. You don't need to set the AFL env vars for Mayhem (`AFL_SKIP_CPUFREQ` and `AFL_I_DONT_CARE_ABOUT_MISSING_CRASHES`) as we will handle this.

File: Mayhemfile [hunk 2]
```diff
@@ -0,0 +1,11 @@
+image: ghcr.io/capuanob/epub2txt2-fuzz:latest
```
Comment (xansec) [inline Mayhemfile]: Please omit image.

File: src/epub2txt.c [hunk 3]
```diff
@@ -435,7 +435,7 @@ void epub2txt_do_file (const char *file, const Epub2TxtOptions *options,
```
Comment (xansec) [inline src/epub2txt.c]: Can you comment on this change?

File: src/main.c [hunk 4]
```diff
@@ -12,6 +12,7 @@
 #include <sys/ioctl.h>
```
Comment (xansec) [inline src/main.c]: Can you also comment on this change?

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

**Comment by capuanob** [comment]
@xansec I think those extraneous changes were uncaught artifacts of my testing prior to you guys introducing disabling smoke tests (zip would fail on the initial test case). I have reset those files to origin/master and made all other change requests.

**Review by xansec — APPROVED** [review]
LGTM!

**Pull request merged**